src/js/angular/repositories/templates/ontop-repo.html

```diff
@@ -57,14 +57,21 @@ <h4>{{'ontop.repo.configuration' | translate}}</h4>
             {{'ontop.repo.properties.label' | translate}}
         </label>
         <div class="col-xs-8 col-md-5 col-lg-4">
-            <div class="mb-1" uib-tooltip="{{'ontop.repo.properties.tooltip' | translate}}">
+            <div class="mb-1" gdb-tooltip="{{'ontop.repo.properties.tooltip' | translate}}">
                 <textarea class="form-control" rows="3" maxlength="1000"
                           id="additionalProperties"
                           ng-model="repositoryInfo.params.additionalProperties.value"
                           placeholder="{{'ontop.repo.properties.placeholder' | translate}}">
                 </textarea>
             </div>
         </div>
+        <div class="col-xs-12 col-md-4">
+            <a class="uri"
+               href="{{ontoProperiesLink}}"
+               style="text-decoration: underline;">
```

Comment:
umm why? Aren't links underlined by default? Or it has something to do with onto's visual theme?